index.html

```diff
@@ -2395,7 +2407,13 @@ <h2>Hypermedia Controls Vocabulary Definitions</h2>
           response message for the primary response.</p><table class="def"><thead><tr><th><a>Vocabulary term</a></th><th>Description</th><th>Assignment</th><th>Type</th></tr></thead><tbody><tr class="rfc2119-table-assertion" id="td-vocab-contentType--ExpectedResponse"><td><code>contentType</code></td><td>Assign a content type based on a media type
                 (e.g., <code>text/plain</code>) and potential
                 parameters (e.g., <code>charset=utf-8</code>) for
-                the media type [<cite><a class="bibref" data-link-type="biblio" href="#bib-rfc2046" title="Multipurpose Internet Mail Extensions (MIME) Part Two: Media Types">RFC2046</a></cite>].</td><td>mandatory</td><td><a href="http://www.w3.org/TR/2012/REC-xmlschema11-2-20120405/#string"><code>string</code></a></td></tr></tbody></table></section>
+                the media type [<cite><a class="bibref" data-link-type="biblio" href="#bib-rfc2046" title="Multipurpose Internet Mail Extensions (MIME) Part Two: Media Types">RFC2046</a></cite>].</td><td>mandatory</td><td><a href="http://www.w3.org/TR/2012/REC-xmlschema11-2-20120405/#string"><code>string</code></a></td></tr>
+<tr class="rfc2119-table-assertion" id="td-vocab-schema--ExpectedResponse"><td><code>schema</code></td><td>Used to define the output data schema 
```

Comment:
Please clarify why this normative assertion is added.

Reply:
These changes introduce new normative behavior and may have undesired impact.
They should be reviewed and discussed in the group before they are added.

Reply:
see #1324 and https://github.com/w3c/wot-thing-description/issues/1053#issuecomment-959452562

Reply:
this is also reverted and outsourced in a separate PR #1341